### PR TITLE
Fixing linting

### PR DESCRIPTION
**Pull request by brumik**
This PR fixes linting, lots of problems with default values and prop type checking, and also moves the pagination on the savings planner page under the cards, so it is les confusing. Further work probably could make it better, but we were not so nitpicking this time.

### CODE REVIEW

File: src/Components/Toolbar/ToolbarDeleteButton.js [hunk 1]
```diff
@@ -264,20 +249,20 @@ function ToolbarDeleteButton({
...
+// TODO Harpreet Kataria I was not able to figure out the exact type of the
+// deleteDetailsRequests. Cna you please fill in?
+// Same for the delete message
+ToolbarDeleteButton.propTypes = {
+  itemsToDelete: PropTypes.arrayOf(ItemToDelete).isRequired,
+  pluralizedItemName: PropTypes.string,
+  errorMessage: PropTypes.string.isRequired,
+  onDelete: PropTypes.func.isRequired,
+  warningMessage: PropTypes.node,
+  deleteDetailsRequests: PropTypes.array.isRequired,
```
Comment (h-kataria) [inline src/Components/Toolbar/ToolbarDeleteButton.js:263]: looks correct to me, it should be array of requests.

Comment (brumik) [reply src/Components/Toolbar/ToolbarDeleteButton.js:263]: And can you describe what is in the request object? I would like to put here something like
```js
PropTypes.arrayOf(PropTypes.shape(...))
```

File: src/Components/DeleteButton/DeleteButton.js [hunk 2]
```diff
@@ -144,14 +147,21 @@ function DeleteButton({
...
+  name: PropTypes.string.isRequired,
+  variant: PropTypes.object.isRequired,
+  isDisabled: PropTypes.bool.isRequired,
   ouiaId: PropTypes.string,
-};
-
-DeleteButton.defaultProps = {
-  ouiaId: null,
+  deleteMessage: PropTypes.string.isRequired,
+  deleteDetailsRequests: PropTypes.array.isRequired,
```
Comment (h-kataria) [inline src/Components/DeleteButton/DeleteButton.js:162]: LGTM, it is expected to be an array of request objects

**Review by h-kataria — APPROVED** [review]

File: src/Components/DeleteButton/DeleteButton.js [hunk 3]
```diff
@@ -10,31 +10,33 @@ import ErrorDetail from '../ErrorDetail';
...
+
+const DeleteButton = ({
   onConfirm,
-  modalTitle,
-  name,
-  variant,
+  modalTitle = 'Delete',
+  name = '',
+  variant = 'secondary',
+  isDisabled = true,
```
Comment (h-kataria) [inline src/Components/DeleteButton/DeleteButton.js:25]: this should be dafaulted to `false` sorry...my bad

**Pull request merged**